plugin/db/mysql/restore.go

```diff
@@ -533,6 +539,21 @@ func (driver *Driver) downloadBinlogFilesOnServer(ctx context.Context, binlogFil
 
 // FetchAllBinlogFiles downloads all binlog files on server to `binlogDir`.
 func (driver *Driver) FetchAllBinlogFiles(ctx context.Context) error {
+	// Ensure that there's at most one ongoing downloading process for the current MySQL instance.
+	muBinlogFile.Lock()
+	if _, ok := binlogFileTasks[driver.connectionCtx.InstanceName]; ok {
```

Comment:
[InstanceName is not UNIQUE](https://sourcegraph.com/github.com/h3n4l/bytebase/-/blob/store/migration/prod/LATEST.sql?L271%3A24=).